flit/inifile.py

```diff
@@ -117,7 +117,12 @@ def read_pkg_ini(path):
     if 'description-file' in md_sect:
         description_file = path.parent / md_sect.pop('description-file')
         with description_file.open() as f:
-            md_dict['description'] = f.read()
+            raw_desc =  f.read()
+        from readme.rst import render
+        _, ok = render(raw_desc)
+        if not ok:
+            raise ValueError("The file description seem to not be valid rst for PyPI")
```

Comment:
> Descriptions don't have to be rst - plain text is valid, that's why it silently falls back to unrendered text if there's an error producing rst.

Plain text would mostly not raise errors, but ok to log.warn.



> Also, it would be good to indicate why it's not valid rst. I think the code from distutils gave a list of error messages. Actually, I think I'd rather copy the distutils code than create unnecessary dependencies on bleach and pygments (though they're a fitting pairing...) through readme.

I understand, though I don't want to get into the buisness of replicating pipy logic in here. I can vendorize [readme/rst.py](https://github.com/pypa/readme/blob/master/readme/rst.py) that does not seem to require any of bleach or pygments.